pallets/liquidity-pools/src/lib.rs

```diff
@@ -243,6 +243,19 @@ pub mod pallet {
 
 		#[pallet::constant]
 		type TreasuryAccount: Get<Self::AccountId>;
+
+		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
+	}
+
+	#[pallet::event]
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	pub enum Event<T: Config> {
+		/// An incoming LP message was
+		/// detected and is further processed
+		IncomingMessage {
+			sender: DomainAddress,
+			msg: MessageOf<T>,
+		},
```

Comment:
On a note: I would like to introduce the opposite `OutgoingMessage` enum event in #1473

Reply:
Should I also add it here do you mean? In this PR?

Reply:
No, not at all. I should have been more explicit. I wanted to express I am happy the opposite of my planned `Event` impl for Liquidity Pools will be implemented by this PR as this is a good signal for my plans.